docs/data-download.md

```diff
@@ -12,6 +12,152 @@ Otherwise `--exchange` becomes mandatory.
     If you already have backtesting data available in your data-directory and would like to refresh this data up to today, use `--days xx` with a number slightly higher than the missing number of days. Freqtrade will keep the available data and only download the missing data.
     Be carefull though: If the number is too small (which would result in a few missing days), the whole dataset will be removed and only xx days will be downloaded.
 
+### Usage
+
+```
+usage: freqtrade download-data [-h] [-v] [--logfile FILE] [-V] [-c PATH] [-d PATH] [--userdir PATH] [-p PAIRS [PAIRS ...]]
+                               [--pairs-file FILE] [--days INT] [--dl-trades] [--exchange EXCHANGE]
+                               [-t {1m,3m,5m,15m,30m,1h,2h,4h,6h,8h,12h,1d,3d,1w} [{1m,3m,5m,15m,30m,1h,2h,4h,6h,8h,12h,1d,3d,1w} ...]]
+                               [--erase] [--data-format {json,jsongz}] [--data-format-trades {json,jsongz}]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -p PAIRS [PAIRS ...], --pairs PAIRS [PAIRS ...]
+                        Show profits for only these pairs. Pairs are space-separated.
+  --pairs-file FILE     File containing a list of pairs to download.
+  --days INT            Download data for given number of days.
+  --dl-trades           Download trades instead of OHLCV data. The bot will resample trades to the desired timeframe as specified as
+                        --timeframes/-t.
+  --exchange EXCHANGE   Exchange name (default: `bittrex`). Only valid if no config is provided.
+  -t {1m,3m,5m,15m,30m,1h,2h,4h,6h,8h,12h,1d,3d,1w} [{1m,3m,5m,15m,30m,1h,2h,4h,6h,8h,12h,1d,3d,1w} ...], --timeframes {1m,3m,5m,15m,30m,1h,2h,4h,6h,8h,12h,1d,3d,1w} [{1m,3m,5m,15m,30m,1h,2h,4h,6h,8h,12h,1d,3d,1w} ...]
+                        Specify which tickers to download. Space-separated list. Default: `1m 5m`.
+  --erase               Clean all existing data for the selected exchange/pairs/timeframes.
+  --data-format {json,jsongz}
```

Comment:
this should probably better be called `--data-format-ohlcv`, to have it aligned with `dataformat_ohlcv` setting...